torch/distributed/elastic/rendezvous/c10d_rendezvous_backend.py

```diff
@@ -166,6 +167,23 @@ def _create_tcp_store(params: RendezvousParameters) -> TCPStore:
 
     return store
 
+def _create_file_store(params: RendezvousParameters) -> FileStore:
+    # The directory is readable, writable, and searchable only by the
+    # creating user ID. Same goes for the created file.
+    # TODO: Right now, this temporary directory and file does not clean itself
+    # up after use. In the future, a close method for rendezvous backend should
+    # be implemented to enable cleaning up resources. (For now, since it
+    # writes to /tmp, it is ok)
+    temp_dir = tempfile.mkdtemp(prefix="torch_rdzv_")
```

Comment:
I would simplify these two lines just to a `_, path = tempfile.mkstemp()`. This is an implementation specific temporary binary file that gets deleted at the end of the job. No need to have a directory structure or a naming convention.